src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -28,11 +28,11 @@
 from avocado import main
 from apricot import TestWithServers
 
-import write_host_file
 from daos_api import DaosApiError
 from conversion import c_uuid_to_str
 from general_utils import get_pool, get_container
 
+# pylint: disable=broad-except
```

Comment:
Really not a fan of this.  The `except` line should be fixed, not the warning skipped.